stories/s3slstr.stories.js

```diff
@@ -192,7 +187,7 @@ export const findTiles = () => {
 };
 
 export const findFlyovers = () => {
-  const layer = new S3SLSTRLayer(instanceId, layerId, null, null, null, null, null, 60, 'NADIR');
+  const layer = new S3SLSTRLayer({ instanceId, layerId, maxCloudCoverPercent: 60, view: 'NADIR' });
```

Comment:
Would it be better to use `view.NADIR` (if available - similar to `orbitDirection` and others in S1 GRD layers) instead of using string?

Reply:
Good idea, thx, fixed.

Reply:
Nice, but here on the line 189 it's still a string.

Reply:
Thx, overlooked, fixed.

Reply:
Great, resolving.